cpp/ps0003/problem 3 s.cpp

```diff
@@ -1,14 +1,18 @@
-#include<iostream>
+#include <iostream>
 using namespace std;
 int main()
 {
-int number;    
-cout <<"Enter any number\n";
-cin >> number;
-if(number % 2 ==0)
-cout << " number is even \n";
-else
-cout << " number is odd \n" << endl;
+  // prompting user for number 2/2
+  int number;    
+  cout << "Enter any number\n";
+  cin >> number;
```

Comment:
- [x] declared a variable to store users input
- [x] used `std::cin` to prompt user for the number